cmd/gossamer/configcmd.go

```diff
@@ -93,8 +93,11 @@ func makeNode(ctx *cli.Context, gen *genesis.GenesisState) (*dot.Dot, *cfg.Confi
 	srvcs = append(srvcs, apiSrvc)
 
 	// RPC
-	fig.RpcCfg = setRpcConfig(ctx, fig.RpcCfg)
-	rpcSrvr := rpc.NewHttpServer(apiSrvc.Api, &json2.Codec{}, fig.RpcCfg)
+	var rpcSrvr *rpc.HttpServer
+	if ctx.GlobalBool(utils.RpcEnabledFlag.Name) {
```

Comment:
I know it's a small check but we don't do any flag checks within `makeNode`, I think it would look cleaner and symmetrical to move this outside of `makeNode` despite it being so small.

Reply:
good call. done!